test/support/movie_server.ex

```diff
@@ -59,7 +63,15 @@ defmodule PropCheck.Test.MovieServer do
     :ok
   end
 
-  def handle_call(:popcorn, _from, s), do: {:reply, :bon_appetit, s}
+  defp popcorn_answer(%__MODULE__{popcorn_failure: false}), do: :bon_appetit
+  defp popcorn_answer(%__MODULE__{popcorn_failure: true}), do:
+    Enum.random([:bon_appetit, :bon_appetit, :bon_appetit,
+      :bon_appetit, :bon_appetit, :bon_appetit,
+      :bon_appetit, :bon_appetit, :bon_appetit,
+      :fuck_off])
```

Comment:
I guess for my _at-home-idea-testing_ phase I could get away with this, but for production code we might be a little family friendly here, and replace this atom ;)

Reply:
I will see, what the google translator will give me back, so please excuse my french ;-)

Reply:
Done, it is now satisfying family values.